.travis.yml

```diff
@@ -90,6 +93,13 @@ matrix:
         - *env_osx_openssl
```

Comment:
`SKIP_PANTSD_TESTS=true` above in the fold is now cruft.

Reply:
Negative. We are still testing pantsd:

https://github.com/pantsbuild/setup/blob/000271094eea1bc3b00dbde8646a4401e09b6af2/tests/test_sanity_check.py#L30-L33